guides/request-validation-csharp/example-1/example-1.4.x.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Configuration;
+using System.Web;
+using System.Net;
+using System.Web.Mvc;
+using Twilio;
+using Twilio.TwiML;
+
+namespace helloworldcsharp
+{
+	public class RequestValidatorFilterAttribute : ActionFilterAttribute
```

Comment:
Additionally, I suggest to apply `AttributeUsage` to this class to limit the application scope for this attribute to only methods, e.g.:

```csharp
[AttributeUsage(AttributeTargets.Method)]
public class ValidateTwilioRequestAttribute : ActionFilterAttribute
{
    // ...
```